lxd/db/cluster/entities.go

```diff
@@ -188,6 +188,46 @@ func (e EntityType) IsNone() bool {
 	return e.entityType == nil
 }
 
+// entityTypes is a list of all instantiated entity types.
+var entityTypes = []EntityType{
+	EntityTypeContainer,
```

Comment:
Cann we avoid exporting these specific implementation types and instead rely on `EntityTypeFromName` to return them? This would ensure we're using the interface properly everywhere.

Reply:
Sounds good. I was following the same pattern as in `shared/entity` but I guess in this case `cluster.EntityTypeContainer` could be reassigned to something else, whereas the types in `shared/entity` couldn't be.